nubis/puppet/consul.pp

```diff
@@ -36,11 +36,9 @@
     group            => 'root',
 }
 
-package { "tar":
```

Comment:
That kind of sucks that its not obvious, I would at least write down a note or a comment up top to say where `package { 'tar': }` is coming from